themes/10up-theme/includes/core.php

```diff
@@ -104,6 +108,10 @@ function scripts() {
 		);
 	}
 
+	/**
+	 * Enqueuing shared.js is required to get css hot reloading working in the frontend
+	 * If you're not shipping any shared js wrap this enqueue in a SCRIPT_DEBUG check.
+	 */
 	/*
```

Comment:
@nicholasio I think you forgot to uncomment this code here.

Reply:
So the shared style were already commented out in the scaffold so I left commented out.